### ISSUE TITLE
[12.0][IMP] base_tier_validation_formula

### ISSUE THREAD

**Issue by ps-tubtim** [ecosoft-odoo/kanban#75]
[12.0][IMP] base_tier_validation_formula, ability to use both Domain and Formula.
https://github.com/OCA/server-ux/issues/114

**Comment by kittiu** [comment]
Yes, better to finish this and get it merged as soon as we can. So, we can finish up the helpers too. :+1:

**Comment by ps-tubtim** [comment]
PR: https://github.com/OCA/server-ux/pull/116
